probe.go

```diff
@@ -409,7 +510,7 @@ func probeInterfaces(c FortiHTTP, registry *prometheus.Registry) bool {
 		Name      string
 		Alias     string
 		Link      bool
-		Speed     int
+		Speed     float64
```

Comment:
Could you modify one of the interfaces in the test file (`testdata/interface.jsonnet`) to return a float number and add a comment in the test file that some versions appear to use floats?